pkg/core/storage/boltdb_store.go

```diff
@@ -107,10 +107,31 @@ func (s *BoltDBStore) PutChangeSet(puts map[string][]byte) error {
 	})
 }
 
+// SeekGC implements the Store interface.
+func (s *BoltDBStore) SeekGC(rng SeekRange, keep func(k, v []byte) bool) error {
+	return boltSeek(s.db.Update, rng, func(c *bbolt.Cursor, k, v []byte) bool {
+		if !keep(k, v) {
+			if c.Delete() != nil {
+				return false
```

Comment:
Then if some key is failed to be deleted `boltSeek` will just stop iteration and return no error. Maybe we'd better return this error from `SeekGC`?

Reply:
Fixed.